test/functional/eval/exists_spec.lua

```diff
@@ -0,0 +1,12 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
I moved this into `eval/environ_spec.lua`. Our test files have too much granularity, and really the common theme for these tests is "environment", not the particular VimL function. (Maybe should live in `core/environ_spec.lua` instead.)